dev/scaletesting/codehostcopy/runner.go

```diff
@@ -18,6 +18,8 @@ import (
 	"github.com/sourcegraph/sourcegraph/lib/output"
 )
 
+const Unlimited = 0
```

Comment:
```suggestion
const NoLimit = 0
```
Hindsight is 20/20 ... this better 😆